docs/usage/cli.rst

```diff
@@ -116,26 +116,26 @@ Options
 | ``-p``\ , ``--port``                  | ``LITESTAR_PORT``               | Bind the the server to this port [default: 8000]               |
 +---------------------------------------+---------------------------------+----------------------------------------------------------------+
-| ``-wc``\ , ``--web-concurrency``      | ``WEB_CONCURRENCY``             | The number of concurrent web workers to start [default: 1]     |
+| ``-W``\ , ``--web-concurrency``      | ``WEB_CONCURRENCY``             | The number of concurrent web workers to start [default: 1]     |
 +---------------------------------------+---------------------------------+----------------------------------------------------------------+
-| ``--host``                            | ``LITESTAR_HOST``               | Bind the server to this host [default: 127.0.0.1]              |
+| ``-H``\ , ``--host``                            | ``LITESTAR_HOST``               | Bind the server to this host [default: 127.0.0.1]              |
 +---------------------------------------+---------------------------------+----------------------------------------------------------------+
-| ``--fd``\ , ``--file-descriptor``     | ``LITESTAR_FILE_DESCRIPTOR``    | Bind to a socket from this file descriptor.                    |
+| ``-F``\ , ``--file-descriptor``      | ``LITESTAR_FILE_DESCRIPTOR``    | Bind to a socket from this file descriptor.                    |
 +---------------------------------------+---------------------------------+----------------------------------------------------------------+
-| ``--uds``\ , ``--unix-domain-socket`` | ``LITESTAR_UNIX_DOMAIN_SOCKET`` | Bind to a UNIX domain socket.                                  |
+| ``-U``\ , ``--unix-domain-socket``  | ``LITESTAR_UNIX_DOMAIN_SOCKET`` | Bind to a UNIX domain socket.                                  |
 +---------------------------------------+---------------------------------+----------------------------------------------------------------+
-| ``--debug``                           | ``LITESTAR_DEBUG``              | Run the application in debug mode                              |
+| ``-d``\ , ``--debug``                 | ``LITESTAR_DEBUG``              | Run the application in debug mode                              |
 +---------------------------------------+---------------------------------+----------------------------------------------------------------+
-| ``--pdb``                             | ``LITESTAR_PDB``                | Drop into the Python debugger when an exception occurs         |
+| ``-P``\ , ``--use_pdb``             | ``LITESTAR_PDB``                | Drop into the Python debugger when an exception occurs         |
 +---------------------------------------+---------------------------------+----------------------------------------------------------------+
 
 
-``--reload-dir``
-++++++++++++++++
+``-R`` | ``--reload-dir``
```

Comment:
I would leave this as is. We don't include the shorthand for the other options either.